docs/configuration/extract-codacy-logs.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
+#### Helper functions ####
+
+usage()
+{
+  echo "Usage: $0 -n NAMESPACE "
+  exit 2
+}
+
+set_variable()
+{
+  local varname=$1
+  shift
+  if [ -z "${!varname}" ]; then
+    eval "$varname=\"$@\""
+  else
+    echo "Error: $varname already set"
+    usage
+  fi
+}
+
+cleanup()
+{
+	if [ -z $LOGS_DIR ]; then
+		rm -r $LOGS_DIR &>/dev/null
+	fi
+}
+
+
+#### Log extraction script ####
+
+while getopts "n:" option; do
+	case ${option} in
+		n )
+			set_variable NAMESPACE ${OPTARG}
+			;;
+		\? )
+			usage
+			;;
+	esac
+done
+
+[ -z "$NAMESPACE" ] && usage
+
+CURRENT_DATE_TIME=$(date "+%Y%m%d-%H%M%S")
+LOGS_DIR=/tmp/codacy_logs_$CURRENT_DATE_TIME
+
+echo "Starting log files extraction"
+
+# Check if kubectl is available
+echo "Checking if kubectl is installed..."
+kubectl version &>/dev/null
+if [ $? -ne 0 ]; then
```

Comment:
You can check the outcome directly like this, maybe yours is more explicit.
```suggestion
if ! kubectl version &>/dev/null; then
```